egs/wsj/s5/steps/data/gen_wav_sample.py

```diff
@@ -0,0 +1,108 @@
+#encoding:utf8
+# Copyright     2017  Ye Bai
+# Apache 2.0
+#
```

Comment:
it's good if you tested it, but please don't check in the testing script.

Reply:
This script was tested ok.

Reply:
Please don't check in this gen_wav_sample.py script.